src/components/navigation/Navigation.js

```diff
@@ -14,7 +14,7 @@ const Navigation = () => {
 
   return (
     <nav>
-      <ul className='flex gap-[54px] text-white'>
+      <ul className='flex justify-between text-white sm:gap-[54px]'>
```

Comment:
I added in my PR responsive navigation! I think after my PR this will be conflict!